src/Keboola/DbWriter/Snowflake/Application.php

```diff
@@ -44,7 +44,7 @@ public function runAction(): string
                 $writer = $this['writer'];
                 $writer->createForeignKeys($table);
             }
-        } elseif ($this['parameters']['export']) {
+        } elseif (!isset($this['parameters']['export']) || $this['parameters']['export']) {
```

Comment:
sorry, nechapu, `!export || export`?

Reply:
!isset(export) || export === true

Reply:
aha, takze export je default true.  ok.

Reply:
kdyz export je default true, nedava to smysl to mit takhle v config definition?

Reply:
to tam je:
https://github.com/keboola/db-writer-snowflake/blob/master/src/Keboola/DbWriter/Snowflake/Configuration/ConfigRowDefinition.php#L68
a stejně to Jakubovi hlasí při testování config row `undefined index export` tak jsem to chtěl zkusit ještě takto - já jsem nepřišel na to jak to nasimulovat

Reply:
to mě tak napadá jestli to nemohlo být tím `$configDefinition = new ConfigRowDefinition;`